src/main/java/service/impl/ReadFileImpl.java

```diff
@@ -0,0 +1,38 @@
+package service.impl;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import service.ReadFile;
+
+public class ReadFileImpl implements ReadFile {
+    private static final String FILE_NAME = "src/main/resources/Input.csv";
```

Comment:
```suggestion
    private static final String FILE_NAME = "src/main/resources/input.csv";
```
тоже давай передавать через параметры метода